docs/guides/setup/NCI/basics.rst

```diff
@@ -106,9 +107,9 @@ repository <https://github.com/GeoscienceAustralia/dea-notebooks/tree/stable>`_
 into your home directory (e.g. :file:`~/dea-notebooks`).
 
 .. note:: DEA Notebooks is a large repository that will take up a large proportion of
-   of available storage space in your home directory. We recommend cloning a new copy 
+   available storage space in your home directory. We recommend cloning a new copy 
    of the repository to a location on ``/g/data/`` when possible. (Learn how to `clone
-   a DEA Notebook <https://github.com/GeoscienceAustralia/dea-notebooks/wiki/Edit-a-DEA-Notebook>`_.)
+   DEA Notebooks <https://github.com/GeoscienceAustralia/dea-notebooks/wiki/Edit-a-DEA-Notebook>`_.)
```

Comment:
Suggestion:

I usually write these 'Learn more' links in the singular form since it is conceptually slightly simpler than plural form.

Reply:
In this case, we're pointing users to the instructions so they can learn how to clone the "DEA Notebooks" repo as a whole, not cloning a single notebook (i.e. using "DEA Notebooks" as a proper noun).

Reply:
Oh, I see. Ok I would word it as: `clone the DEA Notebooks repository`